ci-operator/step-registry/telcov10n/functional/trigger-job/telcov10n-functional-trigger-job-commands.sh

```
@@ -0,0 +1,53 @@
+  echo "🛑 MULTISTAGE_PARAM_OVERRIDE_SKIP_CHAIN_TRIGGER=true — skipping script"
+  exit 0
+fi
+
+echo "Set trigger TOKEN env var"
+TOKEN=$(cat /var/prow-trigger-token/token)
+
+for attempt in $(seq 1 "${MAX_RETRIES}"); do
+  echo "Attempt ${attempt}/${MAX_RETRIES}: triggering job..."
+
+  # Make a request and store the HTTP status code
+  http_code=$(curl -s -o "${ARTIFACT_DIR}/trigger_job.log" -w '%{http_code}' -X POST \
```

Shellcheck (0.11.0)

[info] Make sure not to read and write the same file in the same pipeline.

(SC2094)